pkg/api/v1/types.go

```diff
@@ -222,6 +222,8 @@ type VolumeSource struct {
 	Glusterfs *GlusterfsVolumeSource `json:"glusterfs,omitempty" description:"Glusterfs volume that will be mounted on the host machine "`
 	// PersistentVolumeClaimVolumeSource represents a reference to a PersistentVolumeClaim in the same namespace
 	PersistentVolumeClaimVolumeSource *PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim,omitempty" description:"a reference to a PersistentVolumeClaim in the same namespace"`
+	// Metadata represents metadata about the pod that should populate this volume
+	Metadata *MetadataVolumeSource `json:"metadata" description: "Metadata volume that will be mounted on the host machine and it will contain POD metadata "`
```

Comment:
1.  There's an extra space between `description:` and the actual description here, remove it
2.  The description should be "Metadata volume containing information about the pod"